src/linter.js

```diff
@@ -223,9 +223,8 @@ export default class Linter {
           log.info('Retrieving metadata from manifest.json');
           return this.io.getFileAsString(MANIFEST_JSON)
             .then((json) => {
-              var result = new ManifestJSONParser(
-                json, this.collector).getMetadata();
-              return result.metadata;
+              var manifestParser = new ManifestJSONParser(json, this.collector);
```

Comment:
I'm tending to use `let` when needed. Inside a function like this there's no difference between `var` and `let`.